lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -596,6 +596,7 @@ export abstract class BaseQueryBuilderService {
      * @param searchUrl search url to navigate to
      */
     async navigateToSearch(query: string, searchUrl: string) {
+        this.update();
```

Comment:
This line was previously here but I removed it as it was introducing another bug, with it the search execution will be called twice causing some errors in various situations, for example if you search for 'test' you will get the results but then after searching for `zxcv` even though you should not get any results you will get the 'old' search results from previous query, that's why I think we need to find some other solution for this bug